packaging/osx/sharedhost/scripts/postinstall

```diff
@@ -11,6 +11,8 @@ INSTALL_DESTINATION=$2
 chmod 755 $INSTALL_DESTINATION/dotnet
 
 # Add the installation directory to the system-wide paths
+# But first create the directory if it doesn't exist
+mkdir -p /etc/paths.d
```

Comment:
@gkhanna79 not sure I follow the question? But to your other question, I don't really know. I tried to research whether Sierra changed `path_helper` and associated artifacts in any way, but no information was forthcoming.